src/scales/scale.time.js

```diff
@@ -36,6 +36,19 @@ module.exports = function(Chart) {
 		},
 		ticks: {
 			autoSkip: false
+		},
+		majorTicks: {
```

Comment:
Also, defaults `major` tick options should be `major: {}`, no need to duplicate values if exactly the same, the merge process will take care of that (same for `minor`).